web/components/inputs/FcDatePicker.vue

```diff
@@ -9,33 +9,35 @@
       offset-y
       transition="scale-transition">
       <template v-slot:activator="{ attrs, on: onMenu }">
-        <v-text-field
-          v-model="valueFormatted"
-          append-icon="mdi-calendar"
-          offset-y
-          v-bind="{ ...attrs, ...$attrs }"
-          @blur="resetValueFormatted"
-          @input="updateValueFormatted"
-          @click:append="showMenu = !showMenu"
-          v-on="onMenu">
-          <template v-slot:append>
-            <v-tooltip right>
-              <template v-slot:activator="{ on: onTooltip }">
-                <v-icon
-                  aria-label="Select date using calendar"
-                  :color="color"
-                  right
-                  v-on="{
-                    ...onMenu,
-                    ...onTooltip,
-                  }">
-                  mdi-calendar
-                </v-icon>
-              </template>
-              <span>Select date using calendar</span>
-            </v-tooltip>
-          </template>
-        </v-text-field>
+        <div v-bind="attrs">
+          <v-text-field
+            v-model="valueFormatted"
+            append-icon="mdi-calendar"
+            offset-y
+            v-bind="$attrs"
+            @blur="resetValueFormatted"
+            @input="updateValueFormatted"
+            @click:append="showMenu = !showMenu"
+            v-on="onMenu">
+            <template v-slot:append>
+              <FcTooltip right>
+                <template v-slot:activator="{ on: onTooltip }">
+                  <v-icon
+                    aria-label="Select date using calendar"
```

Comment:
we might have to add `:aria-hidden="false"` here

Reply:
nvm, Vuetify adds useful ARIA roles here (probably because this is used as an activator target?)